reason-react-native/src/apis/Style.rei

```diff
@@ -1,5 +1,12 @@
 type t;
 
+external array: array(t) => t = "%identity";
+external arrayOption: array(option(t)) => t = "%identity";
+external list: list(t) => t = "%identity";
+external listOption: list(option(t)) => t = "%identity";
+let unsafeAddStyle: (t, Js.t('a)) => t;
+let unsafeStyle: Js.t('a) => t;
```

Comment:
You have a `let` instead of `external`. Simply replacing with 
```reason
external unsafeStyle: Js.t('a) => t = "%identity";
```
should be enough to resolve

Reply:
That's not an issue. You can define `external thing` in interface as `let thing`. Btw, I changed some for the moved code.

Reply:
Actually it works with let, but create unnecessary code in the js, so I adjusted as you recommended.